lib/accessories/thermostat.js

```diff
@@ -115,31 +127,20 @@ FritzThermostatAccessory.prototype.setTargetHeatingCoolingState = function(state
         return;
     }
 
+    var service = this.services.Thermostat;
```

Comment:
const wenn Du es schon änderst?

Reply:
Okay. Triviale Sache, lässt sich ändern.

Reply:
Beziehungsweise, ich glaube das ist eine Zeile, die durch den rebase doppelt reingerutscht ist sehe ich gerade. Service ist in der ersten Zeile der Funktion bereits definiert.